test/manifests/v1alpha2/configmap.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: descheduler-policy-configmap
+  namespace: kube-system
+data:
+  policy.yaml: |
+    apiVersion: "descheduler/v1alpha2"
+    kind: "DeschedulerPolicy"
+    profiles:
+      - name: ProfileName
+        pluginConfig:
+        - name: "DefaultEvictor"
+          args:
+            priorityThreshold:
+              value: 10000
+        - name: "PodLifeTime"
+          args:
+            maxPodLifeTimeSeconds: 86400
+            namespaces:
+              include:
+              - "namespace1"
+              - "namespace2"
+        plugins:
+          evict:
```

Comment:
Let's wait for https://github.com/kubernetes-sigs/descheduler/pull/1120 to merge and update the extension point accordingaly

Reply:
/hold cancel